### ISSUE TITLE
Duplicate Countries in Filter

### ISSUE THREAD

**Issue by thomersch** [thomersch/openstreetmap-calendar#25]
In the location filter dropdown, the US is listed twice: as "United States" and "United States of America":

<img width="251" alt="Screenshot 2020-03-16 at 20 37 25" src="https://user-images.githubusercontent.com/948605/76793792-fd0e1c00-67c5-11ea-93c1-806d21afc513.png">

We just take display all the available countries in the database in alphabetical order. Those values are coming from Nominatim, thus being highly dependent on consistent tagging in OSM. We should consider whether to implement a synonym list or trying to clean up values somehow.

**Comment by thomersch** [comment]
I cleaned up the database entry, but the issue might arise in the future again. Since the filter now supports ISO codes (e.g. DE, US etc), we could use some fixed list.

**Comment by danieldegroot2** [comment]
@thomersch The issue is currently present. I've also noted "TW" and "Taiwan".
![image](https://user-images.githubusercontent.com/67521919/155175295-eb909000-f182-4f44-9c7f-23f241eeef8f.png)

**Comment by thomersch** [comment]
Yeah, this will pop up again and again, since we're just using OSM data.

I looked whether we could use ISO 3166 lists (e.g. https://github.com/lukes/ISO-3166-Countries-with-Regional-Codes/blob/master/all/all.csv), but I haven't found a consistent one. This one for example says "Moldova, Republic of" and "United Kingdom of Great Britain and Northern Ireland", but most countries are addressed by their shorthand name, e.g. "Germany".

I'd rather also not display just two letter ISO codes, but have proper labels.

Maybe we could get a different data source, like Wikidata or Natural Earth. We don't even need geometries, just sane labels would suffice. If someone could come up with e.g. a Wikidata SPARQL query, this would make me very happy (or at least, less sad).

**Comment by danieldegroot2** [comment]
> In the location filter dropdown, the US is listed twice: as "United States" and "United States of America":
> 
> <img alt="Screenshot 2020-03-16 at 20 37 25" width="251" src="https://user-images.githubusercontent.com/948605/76793792-fd0e1c00-67c5-11ea-93c1-806d21afc513.png">
> 
> We just take display all the available countries in the database in alphabetical order. Those values are coming from Nominatim, thus being highly dependent on consistent tagging in OSM. We should consider whether to implement a synonym list or trying to clean up values somehow.

### Question
What is being used? (an OSM key?)

### Example
https://www.openstreetmap.org/relation/148838

#### United States
* [`name`](https://wiki.openstreetmap.org/wiki/Key:name)
* [`name:en`](https://wiki.openstreetmap.org/wiki/Key:name:en)
* [`wikipedia`](https://wiki.openstreetmap.org/wiki/Key:wikipedia) (can be derived from Wikidata)

#### United States of America
* [`int_name`](https://wiki.openstreetmap.org/wiki/Key:int%20name)
* [`official_name`](https://wiki.openstreetmap.org/wiki/Key:official%20name)
* [`official_name:en`](https://wiki.openstreetmap.org/wiki/Key:official%20name:en)

### Other examples
have not checked other instances of similar name (read: island/islet, boundary etc.)

#### Czechia
https://www.openstreetmap.org/relation/51684
`int_name`, `name:en`

#### DR Congo
https://www.openstreetmap.org/relation/192795
`short_name:en`

#### TW
https://www.openstreetmap.org/relation/449220
`short_name`, `short_name:en`

### Solution(s)
* Use (some) JSONs.
* Filter (Nominatim) to definition "Country" or OSM admin level(s) (ex. 2) / ISO code(s) (in OSM) / something else (in OSM).
* Check how ResultMaps / Community Index / etc. do it.
* Check how non-OSM websites do it.

**Comment by thomersch** [comment]
Currently, osmcal uses the `country` attribute from Nominatim. ([Geocoding code](https://github.com/thomersch/openstreetmap-calendar/blob/69cde6a1838754ec95ffe6ea38a8df58926ad232/osmcal/models.py#L48), [Display code](https://github.com/thomersch/openstreetmap-calendar/blob/69cde6a1838754ec95ffe6ea38a8df58926ad232/osmcal/views.py#L78))

The information about the SQL from Nominatim file seems to be outdated. They've since moved to a different system, but that seems to be quite reasonable. It's a list of all countries, even with translations: https://github.com/osm-search/Nominatim/blob/master/settings/country_settings.yaml

Now we just need a system to get this into osmcal.
